JREProgressView.podspec

```diff
@@ -0,0 +1,21 @@
+Pod::Spec.new do |s|
+  s.name         = "JREProgressView"
+  s.version      = "0.0.1-so.1"
+  s.summary      = "UIProgressView subclass"
+  s.description  = <<-DESC
+                   UIProgressView subclass that supports custom images on iOS 7.1.
+                   DESC
+  s.homepage     = "https://github.com/silverorange/JREProgressView"
+  s.license      = { :type => "MIT", :file => "LICENSE" }
+  s.authors      =  { "Keith Burgoyne"   => "keith@silverorange.com",
+                      "Michael Gauthier" => "mike@silverorange.com",
+                      "John Rommel Estropia" => "" }
+  s.platform     = :ios, "7.1"
+  s.ios.deployment_target = "7.1"
+  s.source       = { :git => "https://github.com/silverorange/JREProgressView.git", :tag => s.version.to_s }
```

Comment:
`https://` github URLs will require username/password auth that will not work with 3rd-party auth enabled Github accounts. Use the SSH git URL instead.